metadata.json

```diff
@@ -69,8 +69,7 @@
     {
       "operatingsystem": "FreeBSD",
       "operatingsystemrelease": [
-        "10",
-        "11"
+        "10"
```

Comment:
that seems a bit odd. Why are you removing 11, but keep 10?

Reply:
Removed both here.